scss/mixins/_forms.scss

```diff
@@ -95,6 +96,14 @@
         // Boosted mod: no box-shadow
 
         // Boosted mod: focus styles don't match validation state
+
+        // Boosted mod: change border size on focus to allow sufficient contrast between error field focused and non focused
+        &:focus {
+          &[data-focus-visible-added] {
+            @include focus-visible($color: $border-color);
```

Comment:
I just pushed a new version with a fix for that. Basically I had to remove a `Boosted mod` to redo what's in Bootstrap:

```diff
- $form-check-input-focus-border:           null !default;
+ $form-check-input-focus-border:           $input-focus-border-color !default;
```
and:
```diff
- $form-check-input-focus-border:           null !default;
+ $form-check-input-focus-border:           $input-focus-border-color !default;
```

Could you please double-check if it's ok for you regarding your comment @louismaximepiton?

Reply:
Fix done. But I am currently wondering if it's a good choice since it will modify focus state of all checkboxes even when they are not on error. I think we need to talk about it with designers. Any opinion and/or comments?